src/vde_backend/sync_sftp.py

```diff
@@ -221,50 +221,50 @@ def getAllHistoryDates(home_id, table_name, now):
 	if len(date_df) > 0:
 		first_date = pd.Timestamp(date_df.iat[0,0])
 
-		all_dates = getDatesBetween(first_date, now)
+		all_dates = get_dates_between(first_date, now)
 
 	return all_dates
 
 
-def processAllHomes(sftp_session, config, default_date, moment, moment_now, now, sftp_info):
+def process_all_homes(sftp_session, config, default_date, moment, moment_now, now, sftp_info):
 	""" 
 	send 1 csv file per home, per day moment (AM or PM) to the sftp server
 	- if no data sent for this home yet, we send the whole history
 	- otherwise, we send data from the last sent date to now
 	"""
 
-	ids = config.getIds()
+	ids = config.get_ids()
 	for home_id in ids.keys():
-		latest_date = getLastDate(sftp_session, home_id)
+		latest_date = get_last_date(sftp_session, home_id)
 		all_dates = [default_date]
 		moments = {default_date: [moment]}
 		if latest_date is None:  # history
-			all_dates = getAllHistoryDates(home_id, TBL_POWER, now)
-			moments = getMoments(all_dates, moment_now)
+			all_dates = get_all_history_dates(home_id, TBL_POWER, now)
+			moments = get_moments(all_dates, moment_now)
 		else:					 # realtime
-			all_dates = getDatesBetween(latest_date, now)
-			moments = getMoments(all_dates, moment_now)
+			all_dates = get_dates_between(latest_date, now)
+			moments = get_moments(all_dates, moment_now)
```

Comment:
As it is a refactor phase, I suggest to put "moments = get_moments(all_dates, moment_now)" outside the if else (code duplicate)

Reply:
We cannot move this line outside the if else because get_moments only applies to those 2 conditions, and the default value is set differently before.